components/cellery-component-test/src/test/resources/testng.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~      http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+-->
+
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="Ballerina-Kubernetes-Test-Suite">
+    <test name="ballerina-kubernetes-sample-tests" parallel="false">
+        <classes>
+            <!--<class name="org.cellery.components.test.HRTest"/>-->
```

Comment:
Is disabling HRTest and EmployeeTest intentional ? Shouldn't we enable them ?

If so please add a TODO comment and a small explanation

Reply:
Yes. It is intentional.
Added in TODO